analytics/views.py

```diff
@@ -1728,6 +1730,9 @@ def by_used_time(row: Dict[str, Any]) -> str:
         "Android",
     ]
 
+    if is_bot:
+        cols.insert(2, "Bot Owner")
```

Comment:
We probably want a `role` column for users who are not a bot owner; maybe worth fixing that while you're working on this?